realm/realm-library/src/main/java/io/realm/internal/datatypes/realminteger/UnmanagedRealmInteger.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.internal.datatypes.realminteger;
+
+import io.realm.RealmInteger;
+
+
+/**
+ * TODO: document the behaviour of this object, as soon as the implementation defines it.
```

Comment:
?  Or make it thread safe, I suppose...  I think that is probably the wrong thing to do.  Your average, garden-variety Pojo is not thread safe...